app/web/i18n/index.ts

```diff
@@ -0,0 +1,7 @@
+import {
+  Trans as nextTrans,
+  useTranslation as nextUseTranslation,
+} from "next-i18next";
+
+export const Trans = nextTrans;
+export const useTranslation = nextUseTranslation;
```

Comment:
```suggestion
export { Trans, useTranslation } from "next-i18next";
```
I think you can also do this to simplify the code a bit!

Reply:
Thanks! Actually I started with that version but this simplified code doesn't work well with VS Code auto-import feature. So, when trying to import useTranslation, it wasn't showing our i18n.ts file as an option, at least in my editor. I prefer to leave this one as it is.

Reply:
I see, it is still coming up but annoyingly not as the first option unless something is already being imported from this file. I'd prefer the simpler syntax... but I guess it's not a big deal breaker for now since the file is relatively simple. Maybe one to review again when we add more complex exports here.